### PR TITLE
Fix node-streams dep; prep next release

### PR DESCRIPTION
**Pull request by JordanMartinez**
**Description of the change**

Fixes `node-streams` dep; preps next release

---

**Checklist:**

- [ ] Added the change to the changelog's "Unreleased" section with a reference to this PR (e.g. "- Made a change (#0000)")
- [ ] Linked any existing issues or proposals that this pull request should close
- [ ] Updated or added relevant documentation
- [ ] Added a test for the contribution (if applicable)

### CODE REVIEW

**Comment by JordanMartinez** [comment]
CC @garyb or @thomashoneyman

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,11 @@ Bugfixes:
...
 Other improvements:
 
+## [v7.0.0](https://github.com/purescript-node/purescript-node-readline/releases/tag/v7.0.0) - 2022-04-29
+
+Breaking changes:
+- Update project and deps to PureScript v0.15.0 (#28 by @JordanMartinez, @sigma-andex)
+
 ## [v6.0.0](https://github.com/purescript-node/purescript-node-readline/releases/tag/v6.0.0) - 2022-04-27
 
 Breaking changes:
```
Comment (thomashoneyman) [inline CHANGELOG.md]: Should this be adjusted to note the error in this release?

Comment (JordanMartinez) [reply CHANGELOG.md]: What error?

Comment (JordanMartinez) [reply CHANGELOG.md]: Oh, yeah. let me fix that.

Comment (JordanMartinez) [reply CHANGELOG.md]: CI builds

**Pull request merged**